EXAMPLE_README.md

```diff
@@ -141,19 +133,22 @@ In order for the app to function correctly, the user must set up their own envir
 # 4️⃣ Testing
 
 🚫Document what you used for testing and why
+- SuperTest
+- Jest
+- Cypress
 
 # 4️⃣ Installation Instructions
 
 🚫explain how to install the required dependencies to get this project up and running with yarn and NPM
+- cd into the main root folder and then type 'npm i'
+- then type 'npm start'
```

Comment:
Remove line 144. And maybe something like this might be cleaner:
Navigate to the root directory with this readme in it.

##### Using NPM:

```console
npm install
```